client/components/scss/Main.scss

```diff
@@ -68,11 +68,11 @@ li.loginSignup {
     padding-left: 0;
 }
 
-#formDiv{
-  text-align: left;
-  padding: 0 12%;
-}
-
-.searchButton {
-  width: 200px;
-}
+// #formDiv{
+//   text-align: left;
+//   padding: 0 12%;
+// }
+
+// .searchButton {
+//   width: 200px;
+// }
```

Comment:
hmm ... we prefer the padding ... otherwise, the search bar is so large and intrusive!  but let's discuss.